xchange-kraken/src/test/java/info/bitrich/xchangestream/kraken/KrakenStreamingTest.java

```diff
@@ -0,0 +1,57 @@
+package info.bitrich.xchangestream.kraken;
+
+import info.bitrich.xchangestream.core.StreamingExchange;
+import info.bitrich.xchangestream.core.StreamingExchangeFactory;
+import io.reactivex.disposables.Disposable;
+import org.junit.Test;
+import org.knowm.xchange.ExchangeSpecification;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class KrakenStreamingTest {
```

Comment:
I don't think this can be a test run by Maven since it'll fail if Kraken is unavailable when a build runs.  Rename to `KrakenManualExample` perhaps and remove the JUnit annotations (as with other exchanges).

Or, better... XChange uses the `...Integration` suffix to make surefire skip tests like this and have them run by failsafe instead, which is excluded from the build by default.  Maybe we should do the same.

Reply:
I have changed this like for other exchange implementations.